content/applications/finance/sign.rst

```diff
@@ -100,6 +100,14 @@ and the status of the signatures.
 .. image:: sign/signature-status.png
    :alt: Signature status
 
+Expiration and reminders
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can set a **validity date** to your document for limited-time offers or send an
```

Comment:
'limited time offers' sounds very, very commercial - so much so that it kind of overpowers the meaning we put into the feature imho
it's also to ensure that requests won't rot in the system forever
perhaps a more generic wording like 'time-sensitive documents' would be better suited